XCUITests/FxScreenGraph.swift

```diff
@@ -38,28 +38,86 @@ let allSettingsScreens = [
     OpenWithSettings,
 ]
 
+let Intro_Organize = "Intro.Organize"
+let Intro_Customize = "Intro.Customize"
+let Intro_Share = "Intro.Share"
+let Intro_Choose = "Intro.Choose"
+let Intro_Sync = "Intro.Sync"
+
+let allIntroPages = [
+    Intro_Organize,
+    Intro_Customize,
+    Intro_Share,
+    Intro_Choose,
+    Intro_Sync,
+]
+
+let HomePanelsScreen = "HomePanels"
+let HomePanel_TopSites = "HomePanel.TopSites.0"
+let HomePanel_Bookmarks = "HomePanel.Bookmarks.1"
+let HomePanel_History = "HomePanel.History.2"
+let HomePanel_ReadingList = "HomePanel.ReadingList.3"
+
+let allHomePanels = [
+    HomePanel_Bookmarks,
+    HomePanel_TopSites,
+    HomePanel_History,
+    HomePanel_ReadingList,
+]
+
+let ContextMenu_ReloadButton = "ContextMenu_ReloadButton"
+
 func createScreenGraph(_ app: XCUIApplication, url: String = "https://www.mozilla.org/en-US/book/") -> ScreenGraph {
-    let map = ScreenGraph(app)
+    let map = ScreenGraph()
 
+    let startBrowsingButton = app.buttons["IntroViewController.startBrowsingButton"]
     map.createScene(FirstRun) { scene in
         scene.gesture(to: NewTabScreen) {
-            let firstRunUI = app.buttons["Start Browsing"]
-            if firstRunUI.exists {
-                firstRunUI.tap()
+            if startBrowsingButton.exists {
+                startBrowsingButton.tap()
             }
         }
+
+        scene.noop(to: allIntroPages[0])
+    }
+
+    // Add the intro screens.
+    var i = 0
+    let introLast = allIntroPages.count - 1
+    let introPager = app.scrollViews["IntroViewController.scrollView"]
+    for intro in allIntroPages {
+        let prev = i == 0 ? nil : allIntroPages[i - 1]
+        let next = i == introLast ? nil : allIntroPages[i + 1]
+
+        map.createScene(intro) { scene in
```

Comment:
I had to go through these screens in the [PR](https://github.com/mozilla-mobile/firefox-ios/pull/2532) for [Navigation Tests](https://github.com/mozilla-mobile/firefox-ios/blob/1cdb14bae8bf6352214f3d36ecc84351c3f38989/XCUITests/NavigationTest.swift#L62), and looking at the new scene here I am wondering a few things...
-Would it be possible to call it also from Settings Screen, from Show Tour? That option, Start tour, goes through this intro. I tried but got an error that cannot route from SettingsScreen to Intro.Organize. Just to know if that could be used in more scenarios not only when the first launch happens
-And once we get to the last screen, not sure if it would make sense to add the scene.tap(SingInToFirefox, to: FxSync). So that can be used also in other tests